tests/pom.xml

```diff
@@ -82,6 +82,12 @@
             <artifactId>spark-sql_${scala.binary.version}</artifactId>
             <version>${spark.test.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-avro_${scala.binary.version}</artifactId>
+            <version>${spark.version}</version>
+            <scope>provided</scope>
```

Comment:
The spark-avro jar dependency should be in the dependency management of the parent pom so we don't need to continually specify the version and scope everywhere (and possibly get it wrong in one or more of those places).